bigquery/samples/utils.py

```diff
@@ -15,8 +15,12 @@
 
 # [START get_service]
 def get_service():
-    from discovery_doc import build_and_update
-    return build_and_update('bigquery', 'v2')
+    from googleapiclient.discovery import build_from_document
+    from oauth2client.client import GoogleCredentials
+    credentials = GoogleCredentials.get_application_default()
+    if credentials.create_scoped_required():
+        credentials = credentials.create_scoped('https://www.googleapis.com/auth/bigquery')
+    return build('bigquery','v2', credentials=GoogleCredentials.get_application_default())
```

Comment:
When I run tox, I get this error:
(python-docs-samples)rennie@rennie:~/gitrepos/python-docs-samples$ tox
py27-nosetest installed: Flask==0.10.1,google-api-python-client==1.4.0,httplib2==0.9.1,itsdangerous==0.24,Jinja2==2.7.3,MarkupSafe==0.23,mock==1.0.1,nose==1.3.6,NoseGAE==0.5.7,oauth2client==1.4.11,pyasn1==0.1.7,pyasn1-modules==0.0.5,rsa==3.1.4,simplejson==3.7.1,six==1.9.0,uritemplate==0.6,Werkzeug==0.10.4
py27-nosetest runtests: PYTHONHASHSEED='4074483713'
py27-nosetest runtests: commands[0] | nosetests --with-gae
EEEEE..S.................
======================================================================
ERROR: test_async_query (bigquery.test.test_async_query.TestAsyncQuery)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/usr/local/google/home/rennie/gitrepos/python-docs-samples/bigquery/test/test_async_query.py", line 28, in test_async_query
    5):
  File "/usr/local/google/home/rennie/gitrepos/python-docs-samples/bigquery/samples/async_query.py", line 48, in run
    service = get_service()
  File "/usr/local/google/home/rennie/gitrepos/python-docs-samples/bigquery/samples/utils.py", line 23, in get_service
    return build('bigquery','v2', credentials=GoogleCredentials.get_application_default())
NameError: global name 'build' is not defined